### PR TITLE
Add AI/ML WG & model signing Q3 report as PR

### PR DESCRIPTION
**Pull request by mihaimaruseac**
This is for preparation for the meeting on Sept 3.

### CODE REVIEW

**Review by sevansdell — APPROVED** [review]
Can't wait to hear the full presentation on the TAC call.  Minor question on line36. What is "wheel"?

**Comment by mihaimaruseac** [comment]
It's the python packaging format (`pip install ${wheel}`). I can clarify in text if needed.

Thank you for the review!

**Review by SecurityCRob — APPROVED** [review]

File: TI-reports/2024/2024-Q3-AI-ML-WG.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Questions/Issues for the TAC
+
+We have one TAC member as co-chair and several other TAC members are frequent participants in both the working group and the model signing project.
+
+We're eager for substantive discussion with TAC and others.
+
+## Additional Information
+
+We are currently in the process of formalizing the lifecycle documents, both for the [working group (sandbox to incubating)](https://github.com/ossf/tac/pull/375) and for the [model signing project (was SIG in previous update)](https://github.com/ossf/tac/pull/347).
```
Comment (marcelamelara) [inline TI-reports/2024/2024-Q3-AI-ML-WG.md:53]: There might be a missing closing "]" for the first link?

**Review by marcelamelara — APPROVED** [review]
LGTM, thanks @mihaimaruseac !

**Review by steiza — APPROVED** [review]

File: TI-reports/2024/2024-Q3-AI-ML-WG.md (hunk 1 above)
Comment (lehors) [reply TI-reports/2024/2024-Q3-AI-ML-WG.md:53]: Seems fine actually.

**Review by lehors — APPROVED** [review]
LGTM

File: TI-reports/2024/2024-Q3-AI-ML-WG.md (hunk 1 above)
Comment (mihaimaruseac) [reply TI-reports/2024/2024-Q3-AI-ML-WG.md:53]: I have `(` `)` in the text of the link, I could move them out to resolve some confusion if that would be helpful

Comment (marcelamelara) [reply TI-reports/2024/2024-Q3-AI-ML-WG.md:53]: Ah. Yeah, you're right, it's all there, I just misread that. Thanks!

**Pull request merged**